### PR TITLE
Added solidity as a subcategory

### LINKED ISSUE #1771: [ADD] Including Solidity as a subcategory within the Language category
### Category Name

Language

### Subcategory Name

Solidity

### Additional Context

There is no subcategory for language solidity which is neccasary for someone who wants to step in blockchain field. To make 
learning process of web3 easier , I want to add some recoureses like some  docs, youtube channel and some web-tutorials  which
will cover solidity from basic to advanced.  

<h2>Resources</h2>
<option>
<ul></ul>
<ul></ul>
</option>

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [ ] I have checked that this category does not exist
- [X] I have checked that this subcategory does not exist
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by TIDYMOUSE**
## Fixes Issue
Closes #1771 

## Changes proposed

1. Added subcategory solidity with links to some of the best resources
2. Made changes following CONTRIBUTING.md
3. Added an ending curly brace on line 207 of CONTRIBUTING.md

## Note to reviewers
Hello, this is my first ever open source contribution. I am open to conversation for any other changes required if any. Thank You.

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @TIDYMOUSE! 😊 Welcome to LinksHub! 😊 I closed your PR for now because you've not been assigned to the issue that it addresses. Please comment your interest in issue [962](https://github.com/rupali-codes/LinksHub/issues/962). If it gets approved, tag me or the other maintainers so we can review it. If it doesn't give approved, refer to our Contributing Guide so you can learn how to contribute to effectively.

**Comment by Anmol-Baranwal** [comment]
@TIDYMOUSE 
Please link the correct issue from next time. It generally confuses us, since either one of us assign you so most are not aware, there is a issue for this.

If you don't link the issue correctly, even after the PR is merged, the issue might stay open, causing huge problems.

I'm doing it for you this time. Please ensure it next time.

Just a heads-up to help you link issue properly next time. Have a great day!

**Comment by TIDYMOUSE** [comment]
Hey @Anmol-Baranwal , thank you for pointing out my mistake. I will definitely keep this in mind from next time. I hope you can review my code soon and tell me if any further corrections are required.

**Review by Anmol-Baranwal — DISMISSED** [review]
@TIDYMOUSE 
Thanks for fixing minor typo in contributing.md.

🚀 Looks good to me.

**Comment by TIDYMOUSE** [comment]
@Anmol-Baranwal Thank You, this was a confidence boost. I hope @rupali-codes and @CBID2 can review this asap.

File: database/languages/solidity.json
Comment (CBID2) [inline database/languages/solidity.json:11]:
```suggestion
    "description": "This site teaches all Solidity concepts with examples from beginner to intermediate level",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @TIDYMOUSE! :) Your PR is off to a great start! There are some things that need improvement. In the future, I suggest running your descriptions through a tool like Grammarly before submitting them as a PR.

**Comment by TIDYMOUSE** [comment]
Done @CBID2 it must have been something I under-looked in search for some good resources.

**Review by CBID2 — CHANGES_REQUESTED** [review]
I added more suggestions. Keep it up!

**Comment by TIDYMOUSE** [comment]
done @CBID2 Anything else?

**Review by CBID2 — APPROVED** [review]
Looks good now @TIDYMOUSE. @Anmol-Baranwal, @rupali-codes, your turn 😃

**Review by rupali-codes — APPROVED** [review]
looks good

**Pull request merged**
